ceaser_cipher.py

```diff
@@ -1 +1,33 @@
+alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+
+def caesar(start_text, shift_amount, cipher_direction):
+  end_text = ""
+  if cipher_direction == "decode":
+    shift_amount *= -1
+  for char in start_text:
+    if char in alphabet:
+      position = alphabet.index(char)
+      new_position = position + shift_amount
+      end_text += alphabet[new_position]
+    else:
+      end_text += char
+
+  print(f"Here's the {cipher_direction}d result: {end_text}")
+
+from art import logo
+print(logo)
+
+
+keep_going = True
+while keep_going == True:
+  direction = input("Type 'encode' to encrypt, type 'decode' to decrypt:\n")
+  text = input("Type your message:\n").lower()
+  shift = int(input("Type the shift number:\n")) % 26
+
+  caesar(start_text=text, shift_amount=shift, cipher_direction=direction)
```

Comment:
As mentioned above, functions fit the overall program if they return values (instead of printing them). The resulting value can then printed by the caller (if that's the required use-case).

As a nice side-effect of this approach, it's possible to have multiple cipher functions that do different kinds of shuffling, and the user could (theoretically) somehow choose between them.